test-suite/testkit/src/server.rs

```diff
@@ -120,10 +122,12 @@ impl Handler<CreateBlock> for TestKitActor {
         let block_info = if let Some(tx_hashes) = msg.tx_hashes {
             let maybe_missing_tx = tx_hashes.iter().find(|h| !self.0.is_tx_in_pool(h));
             if let Some(missing_tx) = maybe_missing_tx {
-                return Err(api::Error::BadRequest(format!(
-                    "Transaction not in mempool: {}",
-                    missing_tx.to_string()
-                )));
+                return Err(api::Error::new(api::HttpStatusCode::BAD_REQUEST)
+                    .title("CreateBlock handler failed.")
```

Comment:
Nit: I'd suggest to change `title` to "Creating block failed" in order to be more understandable.